conf/efficientnets.yaml

```diff
@@ -2,22 +2,17 @@
 name: efficientnet-b0
 defaults:
   - dm: cifar10 # for LightningDataModule
+  - optim: sgd
 
 # for logger, checkpoint and pretrain weights
 logger: true
 ckpt: true
 pretrained: true
+num_classes: 10
 
 # .test() or not
 test: true
 
-# for LightningModule
-lm:
-  optimizer: Adam
-  learning_rate: 0.001
-  weight_decay: 0.05
-  num_classes: 10
-
 # pl.Trainer configs
 pl:
```

Comment:
Better create this one too, named **Trainer**

and 

`
  trainer:
    fast_dev_run: false
    gpus: 1
    max_epochs: 50
    distributed_backend: dp
    accumulate_grad_batches: 1
    profiler: False
    log_save_interval: 100
    gradient_clip_val: 0.5
    num_sanity_val_steps: 0
    limit_train_batches: 1.0
    limit_val_batches: 1.0
    limit_test_batches: 1.0
    weights_summary:
`

Reply:
that is already under pl

Reply:
oh yea, that's right, no need to split folder